### PR TITLE
Several small fixes

### PR DESCRIPTION
**Pull request by cmickeyb**
This PR is more of an appropriate size for @bvavala with several small fixes. It should be independent of the makefiles PR (meaning that they should be able to be applied in either order).

### CODE REVIEW

**Review by bvavala — APPROVED** [review]
(thanks for the consideration)

lgtm

**Review by marcelamelara — COMMENTED** [review]
Addresses #353, so that's nice! Does this PR actually use the latest SGXSDK release?(documentation is at 2.15.1) If so, would probably be good to update the install docs while we're at it.

**Comment by cmickeyb** [comment]
> Addresses #353, so that's nice! Does this PR actually use the latest SGXSDK release?(documentation is at 2.15.1) If so, would probably be good to update the install docs while we're at it.

i don't think i've updated recently. changing the sgx sdk requirements would be separate.

**Comment by marcelamelara** [comment]
#353 Was only about removing the deprecation warnings, so that's still fine. FWIW, the Dockerfile installs whichever SGX PSW version is the latest on 20.04, so we might already be using a higher-version PSW component at least, even if the SDK is pinned at 2.15.1. It all works, but just worth noting.

**Review by marcelamelara — APPROVED** [review]

File: client/bin/pdo-invoke.psh [hunk 1]
```diff
@@ -53,9 +54,17 @@ fi
 trap_error
 
 if -n ${params}
-   send -s _result_ -f ${pdo_file} ${method}
+   if -e ${wait} "yes"
+      send --wait -s _result_ -f ${pdo_file} ${method}
+   else
```
Comment (prakashngit) [inline client/bin/pdo-invoke.psh:59]: Two comments on the wait flag usage:

1. wait_for_completion I believe only waits for local txn commit, is this sufficient for the multi-user tests?
2. if I look at client/controller/commands/send.py, the wait flag does not seem to be used at all. The decision to wait seems to be based on "if update_response.state_changed and commit" and not on the wait flag itself.

Comment (cmickeyb) [reply client/bin/pdo-invoke.psh:59]: Agreed. This does not fix the multi-test problem. That's going to fall into your lap (see the comments in "send"). We really need to re-think the whole CCF submitter & replication architecture.

It is still the right changes. Just need to "finish" them. Future work.

**Review by prakashngit — APPROVED** [review]
changes to send.py look good to me. Thanks.

**Pull request merged**